src/molecules/AnimatedCheckmark/AnimatedCheckmark.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { tokens } from '@equinor/eds-tokens';
+
+import { AnimatedCheckmarkProps } from './AnimatedCheckmark';
+
+import styled, { keyframes } from 'styled-components';
+
+const { colors } = tokens;
+
+const animateCheckmark = keyframes`
+  from {
+    stroke-dasharray: 130;
+    stroke-dashoffset: 130;
+  }
+  to {
+    stroke-dasharray: 130;
+    stroke-dashoffset: 260;
+  }
+`;
+
+interface ContainerProps {
+  $size: AnimatedCheckmarkProps['size'];
+}
+
+export function sizeToPx(size: AnimatedCheckmarkProps['size']): string {
```

Comment:
Hmm. Not sure if this should be moved to utils file since it is so tied to the css. Guess that is why it is here? 😏

Reply:
Pushed new commit